{
    "config": {
        "abort": {
            "already_configured": "Slu\u017eba je ji\u017e nastavena",
            "bad_config": "Po\u017eadavek selhal kv\u016fli \u0161patn\u00e9mu nastaven\u00ed: Zkontrolujte, zda jsou na [stationboard]({stationboard_url}) platn\u00e9 stanice.",
            "cannot_connect": "Nelze se p\u0159ipojit k serveru",
            "unknown": "Nezn\u00e1m\u00e1 chyba byla vyvol\u00e1na python-opendata-transport"
        },
        "error": {
            "bad_config": "Po\u017eadavek selhal kv\u016fli \u0161patn\u00e9mu nastaven\u00ed: Zkontrolujte na [stationboard]({stationboard_url}), zda jsou n\u00e1zvy va\u0161ich stanic platn\u00e9.",
            "cannot_connect": "Nelze se p\u0159ipojit k serveru",
            "too_many_via_stations": "P\u0159\u00edli\u0161 mnoho propojovac\u00edch stanic; na jedno p\u0159ipojen\u00ed je povoleno jen 5 propojovac\u00edch stanic.",
            "unknown": "Nezn\u00e1m\u00e1 chyba byla vyvol\u00e1na python-opendata-transport"
        },
        "step": {
            "user": {
                "data": {
                    "from": "Po\u010d\u00e1te\u010dn\u00ed stanice",
                    "to": "Kone\u010dn\u00e1 stanice",
                    "via": "Seznam a\u017e p\u0159es 5 stanic"
                },
                "description": "Poskytn\u011bte po\u010d\u00e1te\u010dn\u00ed a koncovou stanici pro va\u0161e p\u0159ipojen\u00ed a voliteln\u011b a\u017e p\u0159es 5 stanic.\n\nZkontrolujte na [stationboardu]({stationboard_url}), zda jsou platn\u00e9 stanice.",
                "title": "\u0160v\u00fdcarsk\u00e1 ve\u0159ejn\u00e1 doprava"
            }
        }
    },
    "entity": {
        "sensor": {
            "delay": {
                "name": "Zpo\u017ed\u011bn\u00ed"
            },
            "departure": {
                "name": "Odjezd"
            },
            "departure0": {
                "name": "Odjezd"
            },
            "departure1": {
                "name": "Odjezd +1"
            },
            "departure2": {
                "name": "Odjezd +2"
            },
            "duration": {
                "name": "Doba trv\u00e1n\u00ed"
            },
            "platform": {
                "name": "Platforma"
            },
            "transfers": {
                "name": "P\u0159enosy"
            }
        }
    },
    "exceptions": {
        "cannot_connect": {
            "message": "Nelze se p\u0159ipojit k serveru.\n{error}"
        },
        "config_entry_not_found": {
            "message": "Instance integrace \u0161v\u00fdcarsk\u00e9 ve\u0159ejn\u00e9 dopravy \"{target}\" nebyla nalezena."
        },
        "invalid_data": {
            "message": "Nastaven\u00ed se nezda\u0159ilo pro polo\u017eku {config_title} s neplatn\u00fdmi daty. Zkontrolujte na [stationboardu]({stationboard_url}), zda jsou n\u00e1zvy va\u0161ich stanic platn\u00e9.\n{error}"
        },
        "not_loaded": {
            "message": "{target} nen\u00ed na\u010dten."
        },
        "request_timeout": {
            "message": "\u010casov\u00fd limit p\u0159i p\u0159ipojov\u00e1n\u00ed pro polo\u017eku {config_title}.\n{error}"
        }
    },
    "issues": {
        "deprecated_yaml_import_issue_bad_config": {
            "description": "Konfigurace \u0161v\u00fdcarsk\u00e9 ve\u0159ejn\u00e9 dopravy pomoc\u00ed YAML se odeb\u00edr\u00e1, ale do va\u0161eho nastaven\u00ed YAML byla importov\u00e1na \u0161patn\u00e1 konfigurace.\n\nZkontrolujte na [stationboardu]({stationboard_url}), zda jsou platn\u00e9 stanice.",
            "title": "Po\u017eadavek importu nastaven\u00ed YAML \u0161v\u00fdcarsk\u00e9 ve\u0159ejn\u00e9 dopravy se nezda\u0159il kv\u016fli \u0161patn\u00e9 konfiguraci"
        },
        "deprecated_yaml_import_issue_cannot_connect": {
            "description": "Nastaven\u00ed \u0161v\u00fdcarsk\u00e9 ve\u0159ejn\u00e9 dopravy pomoc\u00ed YAML se odeb\u00edr\u00e1, ale p\u0159i importu va\u0161eho nastaven\u00ed YAML do\u0161lo k chyb\u011b p\u0159ipojen\u00ed.\n\nUjist\u011bte se, \u017ee se Home Assistant m\u016f\u017ee dostat na [opendata server]({opendata_url}). Pokud je server mimo provoz, zkuste to znovu pozd\u011bji.",
            "title": "Import nastaven\u00ed YAML \u0161v\u00fdcarsk\u00e9 ve\u0159ejn\u00e9 dopravy se nem\u016f\u017ee p\u0159ipojit k serveru"
        },
        "deprecated_yaml_import_issue_unknown": {
            "description": "Nastaven\u00ed \u0161v\u00fdcarsk\u00e9 ve\u0159ejn\u00e9 dopravy pomoc\u00ed YAML se odeb\u00edr\u00e1, ale p\u0159i importu va\u0161eho nastaven\u00ed YAML do\u0161lo k nezn\u00e1m\u00e9 chyb\u011b.\n\nZkontrolujte va\u0161e nastaven\u00ed nebo se pod\u00edvejte na dokumentaci integrace.",
            "title": "Import nastaven\u00ed YAML \u0161v\u00fdcarsk\u00e9 ve\u0159ejn\u00e9 dopravy se nezda\u0159il s nezn\u00e1mou chybou vyvolanou python-opendata-transport"
        }
    },
    "services": {
        "fetch_connections": {
            "description": "Z\u00edsk\u00e1 seznam spoj\u016f ze \u0161v\u00fdcarsk\u00e9 ve\u0159ejn\u00e9 dopravy.",
            "fields": {
                "config_entry_id": {
                    "description": "Instance \u0161v\u00fdcarsk\u00e9 ve\u0159ejn\u00e9 dopravy k vyhled\u00e1n\u00ed spojen\u00ed.",
                    "name": "Instance"
                },
                "limit": {
                    "description": "Po\u010det p\u0159ipojen\u00ed k na\u010dten\u00ed z [1\u201315]",
                    "name": "Limit"
                }
            },
            "name": "Na\u010d\u00edst p\u0159ipojen\u00ed"
        }
    }
}